docs/scout.txt

```diff
@@ -0,0 +1,199 @@
+        searchable collection
+      
+      .. tip:: Queueing
+      
+         When using Scout, consider configuring a queue driver to reduce
+         response times for your application's web interface. To learn more,
+         see the `Queuing section
+         <https://laravel.com/docs/{+laravel-docs-version+}/scout#queueing>`__
+         of the Laravel Scout documentation and the :ref:`laravel-queues` guide.
+
+   .. step:: Create the Atlas Search index
+
+      After you configure Scout and set your default search driver, you can
+      create your searchable collection and search index by running the
+      following command from your application's root directory:
+      
+      .. code-block:: bash
+      
+         php artisan scout:index 'App\Models\Movie'
+      
+      Because you set MongoDB as the default search driver, the preceding
+      command creates the search collection with an Atlas Search index in your
+      MongoDB database. The collection is named ``scout_movies``, based on the prefix
+      set in the preceding section. The Atlas Search index is named ``scout``
+      and has the following configuration:
+
+      .. code-block:: json
+      
+         {
+           "mappings": {
+             "dynamic": true
+           }
+         }
+      
+      .. note::
+      
+         MongoDB can take up to a minute to create and finalize
+         an Atlas Search index, so the ``scout:index`` command might not
+         return a success message immediately.
+
+   .. step:: Import data into the searchable collection
+
+      You can use Scout to replicate data from a source collection into a
+      searchable collection. The following command replicates and indexes data
+      from the ``movies`` collection into the ``scout_movies`` collection
+      created in the preceding section:
+      
+      .. code-block:: bash
+      
+         php artisan scout:import 'App\Models\Movie'
+      
+      The documents are automatically indexed for Atlas Search queries.
+
+      .. tip:: Select Fields to Import
+
+         You might not need all the fields from your source documents in your
+         searchable collection. Limiting the fields you replicate can improve
+         your application's speed and performance.
+         
+         You can select specific fields to import by defining the
+         ``toSearchableArray()`` method in your Eloquent model class. The
+         following code demonstrates how to define ``toSearchableArray()`` to
+         select only the ``plot`` and ``title`` fields for replication:
+         
+         .. code-block:: php
+            
+            class Movie extends Model
+            {
+                ....
+                public function toSearchableArray(): array
+                {
+                    return [
+                        'plot' => $this->plot,
+                        'title' => $this->title,
+                    ];
+                }
+            }
+
```

Comment:
S: the end of this page seems a little abrupt, maybe add a final conclusion sentence along the lines of:
```suggestion
After completing these steps, you can use your {+odm-long+} application to run Atlas Search queries on a searchable collection called ``scout_movies``.
```

Reply:
good idea, I was also mulling over what to put here. I was planning on linking out to the still in prog atlas docs in #3255